src/views/ContinuationIn/ContinuationInBusinessBc.vue

```diff
@@ -4,9 +4,13 @@
     <section class="mt-10">
       <header id="company-name">
         <h2>Company Name</h2>
+        <p></p>
       </header>
 
-      <div>** Correct Name component goes here **</div>
+      <v-card flat class="py-8 px-6">
```

Comment:
Does VS Code show you the warnings like the one below?

![image](https://github.com/bcgov/business-create-ui/assets/10002889/e21f677e-fe97-4dd2-9626-32b9639eeef5)

Make sure you have added the following extensions to VS Code:
- ESLint
- Vue - Official

Reply:
I just installed the 2 extensions and updated the formatting. Thank you.